### ISSUE TITLE
Print account when executing `prisma login`

### ISSUE THREAD

**Issue by timsuchanek** [prisma/prisma#2980]
After you logged in with `prisma login`, right now we just print this:

```
Opening https://app.prisma.io/cli-auth?secret=$2a$08VnnPCGntoLaGpghlagoyNQe in the browser

Authenticating ✔

```

I would like to see, to which account I logged in:
```
Opening https://app.prisma.io/cli-auth?secret=$2a$08VnnPCGntoLaGpghlagoyNQe in the browser

Authenticating ✔

Authenticated with mail@example.com

```

**Comment by divyenduz** [comment]
This is fixed in beta version `1.25.0-beta.12`. Please give it a go via `npm install -g prisma@beta` and let me know if that works for you. Thanks.

**Comment by divyenduz** [comment]
This is available in the latest version `1.25.0`, `npm install -g prisma`. I am closing this issue. Please let me know if the issue still persists via opening a new issue.
